src/main/java/com/thirdcc/webapp/web/rest/TransactionResource.java

```diff
@@ -104,6 +105,16 @@ public TransactionResource(TransactionService transactionService, TransactionQue
         HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(ServletUriComponentsBuilder.fromCurrentRequest(), page);
         return ResponseEntity.ok().headers(headers).body(page.getContent());
     }
+    
+    @PutMapping("/transactions/{id}/status/{transactionStatus}")
```

Comment:
Since we just changing the single attribute in transaction, transactionStatus, we should use PATCH instead of PUT.

And is there any reason for using pathVariable instead of RequestBody?